sdk/typescript/dev/node/src/index.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  dag,
+  Container,
+  Directory,
+  object,
+  func,
+  field,
+} from "@dagger.io/dagger"
+
+import { Commands } from "./commands"
+import { withNpm, withPnpm, withYarn } from "./pkgManager"
+
+@object
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+class Node {
+  @field
+  version = "16-alpine"
+
+  // Private field that contains the node container
+  ctr = dag.container().from(`node:${this.version}`)
```

Comment:
It's possible though, but I'm not longer doing that since [4474436](https://github.com/dagger/dagger/pull/6522/commits/4474436f1e8112cf2e70b5f2e8ea0326b27ca936)